Backend/app/kaal-chakra/src/App.hs

```diff
@@ -51,6 +56,25 @@ allocatorHandle flowRt env =
       jobHandlers =
         emptyJobHandlerList
           & putJobHandlerInList (liftIO . runFlowR flowRt env . runDailyJob)
+          & putJobHandlerInList (liftIO . runFlowR flowRt env . runWeeklyJob)
+          & putJobHandlerInList (liftIO . runFlowR flowRt env . runMonthlyJob)
+          & putJobHandlerInList (liftIO . runFlowR flowRt env . runQuarterlyJob)
+    }
+  where
+    setTablesOption = do
+      opt <- L.getOption KBT.Tables
+      when (isNothing opt) $ do
+        logError "KV Tables option did not found, setting up empty value."
+        L.setOption KBT.Tables emptyKVConfigs
```

Comment:
should we fetch from db instead of using empty values here and on line 105?

Reply:
we can.this will be better in case it returns nothing